pkg/plugin/view/kubehunter/report_view.go

```diff
@@ -74,3 +80,25 @@ func createTable(section starboard.KubeHunterOutput) component.Component {
 
 	return table
 }
+
+func NewKubeHunterReportSummary(report *starboard.KubeHunterReport) (summary *component.Summary) {
+	totalHigh := 0
+	totalMedium := 0
+	totalLow := 0
+
+	for _, section := range report.Report.Vulnerabilities {
+		totalHigh += map[bool]int{true: 1, false: 0}[section.Severity == "high"]
+		totalMedium += map[bool]int{true: 1, false: 0}[section.Severity == "medium"]
+		totalLow += map[bool]int{true: 1, false: 0}[section.Severity == "low"]
+		println(section.Severity)
```

Comment:
I think we don't need this print statement

Reply:
my bad ! I forgot it